src/compiler/marks.ts

```diff
@@ -1,13 +1,9 @@
-'use strict';
-
-require('../globals');
-
-var marks = module.exports = {};
-
-marks.def = function(encoding, layout, style) {
+import {X, Y, TEXT, COLOR, SIZE, SHAPE} from '../consts';
```

Comment:
Totally agree. We don't clutter the global namespace any more and the imports are explicit. When we enable more typescript lint rules it will also tell us about unused imports.